src/lib.rs

```diff
@@ -26,11 +26,15 @@ use std::io;
 use std::mem;
 use std::os::unix::io::{AsRawFd, FromRawFd, IntoRawFd, RawFd};
 use std::pin::Pin;
+use std::ptr;
```

Comment:
Can you please fix the `unused_imports` warning on non-Linux?

```suggestion
#[cfg(target_os = "linux")]
use std::ptr;
```

Reply:
Thanks, I foegot to add that to str::ptr